### DISCUSSION TITLE
Converting from Stream

### DISCUSSION THREAD

**Discussion by andreykorolev** [dlemstra/Magick.NET#1030] [Help]
Hello all,

Magick doesn't create /tmp/magick-XXXX temporary file when converting from a file.
But when I convert from a stream it creates temporary file.
How to disable this? I want to use memory but not disk.

**Comment by dlemstra** [comment]
Not all formats support reading from memory, some require a file. What is the format of the image that you are trying to read?

**Reply by andreykorolev** [reply]
CR3, PSD for example

**Comment by dlemstra** [comment]
The `CR3` file format requires readig from a file at the moment. But we are using `libraw` here so I wonder if we could remove this restriction.
